src/vconsole/vconsole-setup.c

```diff
@@ -36,6 +36,148 @@
 #include "terminal-util.h"
 #include "virt.h"
 
+typedef enum VCMeta{
+        VC_KEYMAP,
+        VC_KEYMAP_TOGGLE,
+        VC_FONT,
+        VC_FONT_MAP,
+        VC_FONT_UNIMAP,
+        _VC_META_MAX,
+        _VC_META_INVALID = -EINVAL,
+} VCMeta;
+
+typedef struct Context {
+        char *config[_VC_META_MAX];
+} Context;
+
+#define CONTEXT_NULL ((Context) {})
```

Comment:
Hmm, is this really needed? Currently it is only used for initialization, so a plain `{}` would be sufficient.

Reply:
Yeah, but I am planning to add more components into Context.